src/shared/components/Card/Card.scss

```diff
@@ -1,144 +1,257 @@
 .card {
-    border: 1px solid #ddd;
-    overflow: hidden;
-    margin: 10px;
-    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+  border: 1px solid #ddd;
+  overflow: hidden;
+  margin: 10px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+  background-color: #F9F4EA;
+  width: 245px;
+  margin: 0px;
+
+  &.dish-card {
+    .card-subtitle {
+      height: 100px;
+    }
+  }
+
+  img {
     width: 245px;
-    height: fit-content;
-    background-color: #F9F4EA;
-    
-    img {
-        width: 245px;
-        height: 151px;
+    height: 151px;
+  }
+
+  .card-content {
+    display: flex;
+    padding: 16px;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 16px;
+
+    .card-title {
+      font-size: 18px;
+      font-weight: 400;
+      line-height: normal;
+      letter-spacing: 2.67px;
+      margin: 0px;
     }
-  
-    .card-content {
+
+    .icon-dish {
       display: flex;
-      width: 245px;
-      padding: 16px;
-      flex-direction: column;
-      align-items: flex-start;
-      gap: 16px;
+      width: 30px;
+      height: 24px;
+      padding: 0px 0.183px 1.022px 0px;
+      justify-content: center;
+      align-items: center;
+    }
+
+    .card-subtitle {
+      font-size: 16px;
+      font-weight: 200;
+      line-height: 20px;
+      letter-spacing: 1.97px;
+      margin: 0px;
       text-align: left;
+    }
 
-      .icon-dish{
-        display: flex;
-        width: 30px;
-        height: 24px;
-        padding: 0px 0.183px 1.022px 0px;
-        justify-content: center;
-        align-items: center;
-      }
+    .price-container {
+      display: flex;
+      align-items: center;
+      gap: 8px;
```

Comment:
the gap is 8px, and take notice that when ur setting style that is for desktop and mobile, u don't needs to duplicate to the media to like u did in line 103, because the desktop is taking the default style from the main style, only if different need to set the prop u want.
 so please remove all duplicate style.

Reply:
fixed